### PR TITLE
Format JSON fixtures

### LINKED ISSUE #21: Pretty-print test fixtures
This won't affect how the tests run, but it will make it easier to examine and diff the fixtures.

### PR DESCRIPTION
**Pull request by Palleas**
Fixes #21.

### CODE REVIEW

**Comment by ikesyo** [comment]
The changes to the submodules should  be reverted, that rolls back to older versions / commits. Those were updated in #29 and #30.

**Comment by Palleas** [comment]
Woops, fixed!

File: update-test-fixtures/main.swift [hunk 1]
```diff
@@ -33,7 +33,10 @@ let result = SignalProducer<FixtureType, NSError>(values: Fixture.allFixtures)
                 
                 if changed {
                     try! fileManager.createDirectoryAtURL(dataURL.URLByDeletingLastPathComponent!, withIntermediateDirectories: true, attributes: nil)
-                    data.writeToURL(dataURL, atomically: true)
+
+                    let JSONResponse = try! NSJSONSerialization.JSONObjectWithData(data, options: .AllowFragments)
+                    let formattedData = try! NSJSONSerialization.dataWithJSONObject(JSONResponse, options: .PrettyPrinted)
+                    formattedData.writeToURL(dataURL, atomically: true)
```
Comment (mdiep) [inline update-test-fixtures/main.swift:39]: TIL!

**Comment by mdiep** [comment]
Thanks! This will make things a lot clearer!

**Pull request merged**